index.html

```diff
@@ -487,6 +494,10 @@ <h3>Current</h3>
                   <img src="https://github.com/lilymcmullen.png" style="max-height: 188px;" class="img-fluid img-thumbnail" alt="Lily McMullen headshot">
                   <p class="mt-2 align-text-bottom"><span class="lead"><a href="https://github.com/lilymcmullen" class="stretched-link" target="_blank" rel="noopener noreferrer">Lily McMullen</a></span></p>
                 </div>
+                <div class="col-6 col-md-4 col-lg-3">
+                  <img src="https://github.com/joeybish.png" style="max-height: 188px;" class="img-fluid img-thumbnail" alt="Beautiful Joey headshot">
+                  <p class="mt-2 align-text-bottom"><span class="lead"><a href="https://github.com/joeybish" class="stretched-link" target="_blank" rel="noopener noreferrer">Joey Bish</a></span></p>
```

Comment:
```suggestion
                  <p class="mt-2 align-text-bottom"><span class="lead"><a href="https://github.com/joeybish" class="stretched-link" target="_blank" rel="noopener noreferrer">Joey Bishop</a></span></p>
```
Not sure if you want Joey or Joseph here, but I'm assuming you meant your full last name here.